setup.py

```diff
@@ -27,8 +27,6 @@ def read_version(package):
     'requests>=2.9.1',
     'six>=1.9',
     'inflection>=0.3.1',
-    'pathlib>=1.0.1; python_version < "3.4"',
```

Comment:
This breaks python 2.7, so it will need a couple more months before it can go.

Reply:
There's #1058 already in which is a partial fix for #998